### PR TITLE
Gather pub release

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

**Comment by InoMurko** [comment]
@unnawut any idea how to bump the tag when creating a release?

File: .circleci/config.yml [hunk 1]
```diff
@@ -366,6 +367,15 @@ jobs:
           min_exit_period: 20
       - snapshot:
           min_exit_period: 120
+      - run: 
+          name: Gather all the snapshots and publish a release
+          command: |
+            if [ "$CIRCLE_BRANCH" = "master" ]; then
+              DATA=$(cat /tmp/release | jq -Rs '.')
+              curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST -d "{\"tag_name\": \"v1.0.0\",\"body\": ${DATA} }" "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases"
```
Comment (unnawut) [inline .circleci/config.yml]:
```suggestion
              curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST -d "{\"tag_name\": \"${TAG}.plasma-${CONTRACT_SHA}\",\"body\": ${DATA} }" "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases"
```

> any idea how to bump the tag when creating a release?

@InoMurko Something like this?

Comment (InoMurko) [reply .circleci/config.yml]: Oh, this is great 👍🏻

**Review by unnawut — APPROVED** [review]
Never try, never know.

**Pull request merged**